src/client/uilogin.js

```diff
@@ -111,7 +111,7 @@ UILogin.doUpdate = function(msPerTick, camera) {
     this.loginButton.stance = 'normal';
 
     if (this.activeButton === this.loginButton) {
-      UICommon.playMouseHoverAudio();
+      UICommon.playMouseHoverAudio(1);
```

Comment:
`UICommon.playMouseHoverAudio` doesn't take any arguments. Not sure why you're passing 1 to it.

Reply:
Probably forgot to remove, I was using during the test phase